This patch is originally from Fedora, thus the following comment. - wally 2011-12-05

This is merely what's required to get it to build, and caribou (the one
thing in Fedora that requires pyclutter) doesn't use this type, so it
should be safe.  But please do drop this patch once pyclutter upstream
catches up with new clutter. - ajax 2011-11-28

diff -up pyclutter-1.3.2/clutter/clutter-base-types.defs.jx pyclutter-1.3.2/clutter/clutter-base-types.defs
--- pyclutter-1.3.2/clutter/clutter-base-types.defs.0002~	2010-10-25 01:00:14.000000000 +0200
+++ pyclutter-1.3.2/clutter/clutter-base-types.defs	2012-03-07 01:28:42.000000000 +0100
@@ -29,12 +29,6 @@
   (vtable "ClutterAnimatableIface")
 )
 
-(define-interface StageWindow
-  (in-module "Clutter")
-  (c-name "ClutterStageWindow")
-  (gtype-id "CLUTTER_TYPE_STAGE_WINDOW")
-  (vtable "ClutterStageWindowIface")
-)
 
 ; boxed types...
 
@@ -357,17 +351,6 @@
   )
 )
 
-(define-enum ShaderError
-  (in-module "Clutter")
-  (c-name "ClutterShaderError")
-  (gtype-id "CLUTTER_TYPE_SHADER_ERROR")
-  (values
-    '("no-asm" "CLUTTER_SHADER_ERROR_NO_ASM")
-    '("no-glsl" "CLUTTER_SHADER_ERROR_NO_GLSL")
-    '("compile" "CLUTTER_SHADER_ERROR_COMPILE")
-  )
-)
-
 (define-enum ScriptError
   (in-module "Clutter")
   (c-name "ClutterScriptError")
